be1-go/hub/standard_hub/mod_test.go

```diff
@@ -821,7 +821,7 @@ func Test_Handle_Answer(t *testing.T) {
 		c: &fakeChannel{},
 	}
 
-	hub, err := NewHub(keypair.public, "", "", nolog, fakeChannelFac.newChannel)
+	hub, err := NewHub(keypair.public, "", "", yeslog, fakeChannelFac.newChannel)
```

Comment:
Since you don't test if the error has been logged or not, I'm not sure why you would enable logs during testing

Reply:
I change that by writing my logs inside a buffer then checking it